workspaces/spectacle/src/spectacle.ts

```diff
@@ -679,6 +679,19 @@ export async function makeSpectacle(opticContext: IOpticContext) {
         };
       },
     },
+    FieldCommands: {
+      remove: (parent: FieldShape) => {
+        return commandGenerator.field.remove(parent.fieldId);
+      },
+      edit: async (
+        parent: FieldShape,
+        args: { requestedTypes: [JsonLike] },
+        context: GraphQLContext
+      ) => {
+        // TODO: query engine for actual commands
+        return [];
+      },
```

Comment:
You'll also want to delete the L676-L697.

I think that commandGenerator.field.remove is also not connected up yet